src/components/molecules/Menu.tsx

```diff
@@ -38,9 +38,10 @@ export default function Menu(): ReactElement {
   return (
     <nav className={styles.menu}>
       <Link to="/" className={styles.logo}>
-        <Logo branding />
+        <Logo noWordmark />
```

Comment:
again have a look at the `branding` implementation please. I think this is required